heim-runtime/tests/linux.rs

```diff
@@ -0,0 +1,31 @@
+use heim_runtime as rt;
+
+#[test]
+#[cfg(target_os = "linux")]
+fn test_default_procfs_root() {
+    let root = rt::linux::procfs_root();
+    assert_eq!("/proc", root);
+}
+
+#[test]
+#[cfg(target_os = "linux")]
+fn test_default_sysfs_root() {
+    let root = rt::linux::sysfs_root();
+    assert_eq!("/sys", root);
+}
+
+#[test]
+#[cfg(target_os = "linux")]
+fn test_nonstandard_sysfs_root() {
+    rt::linux::set_sysfs_root("/host/sys".to_string());
+    let root = rt::linux::sysfs_root();
+    assert_eq!("/host/sys", root);
+}
+
+#[test]
+#[cfg(target_os = "linux")]
+fn test_nonstandard_procfs_root() {
+    rt::linux::set_procfs_root("/host/proc".to_string());
+    let root = rt::linux::procfs_root();
+    assert_eq!("/host/proc", root);
+}
```

Comment:
Tests are run in arbitrary order, so it is possible that the bottom ones are run before the top ones. Since the process space is shared, the tests will fail. I propose to either fork the test process somehow (i.e. https://github.com/AltSysrq/rusty-fork), or split the tests into separate files (this will run them as independent executions).

Reply:
Oh yep. I completely forgot about this. Fixed!